SalesforceLiveAgentApp.ts

```diff
@@ -50,7 +50,7 @@ export class SalesforceLiveAgentApp
 		super(info, logger, accessors);
 	}
 
-	public async initialize(configurationExtend: IConfigurationExtend, environmentRead: IEnvironmentRead): Promise<void> {
+	public async initialize(configurationExtend: IConfigurationExtend, _environmentRead: IEnvironmentRead): Promise<void> {
```

Comment:
Why have we started replacing some of these parameter names with leading underscores? I've noticed it in a few PRs and would like to know why, so I can follow whatever the new convention is.

Reply:
@csmith14  Underscores indicate function parameters that need to exist but are not used within the function block. We avoid TSlint issues that way.